All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -0,0 +1,167 @@
+#####################################################
+# Social Care SDS Data
+# Author: Jennifer Thom
+# Date: July 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Data from Social care database DVPROD
+# Description - Process SDS data
+#####################################################
+
+# Load packages
+
+library(readr)
+library(dplyr)
+library(dbplyr)
+library(phsmethods)
+library(tidyverse)
+library(lubridate)
+
+
+# Set up------------------------------------------------------------------
+
+latest_update <- "Jun_2022"
+
+social_care_dir <- fs::path("/conf/hscdiip/SLF_Extracts/Social_care")
+
+
+# Read Demographic file----------------------------------------------------
+
+sc_demographics <- haven::read_sav(fs::path(
+  social_care_dir,
+  paste0("sc_demographics_lookup_", latest_update),
+  ext = "zsav"
+)) %>%
+  arrange(sending_location, social_care_id)
+
+
+# Query to database -------------------------------------------------------
+
+# set-up conection to platform
+db_connection <- phs_db_connection(dsn = "DVPROD")
+
+# read in data - social care 2 demographic
+sds_full_data <- tbl(db_connection, in_schema("social_care_2", "sds_snapshot")) %>%
+  select(
+    sending_location,
+    social_care_id,
+    period,
+    sds_start_date,
+    sds_end_date,
+    sds_option_1,
+    sds_option_2,
+    sds_option_3
+  ) %>%
+  collect()
+
+# Data Cleaning-----------------------------------------------------
+
+sds_full_clean <- sds_full_data %>%
+  # Match on demographics data (chi, gender, dob and postcode)
+  left_join(sc_demographics, by = c("sending_location", "social_care_id")) %>%
+  # If sds start date is missing, assign start of FY
+  mutate(
+    start_fy = as.Date(paste0(period, "-04-01")),
+    sds_start_date = if_else(is.na(sds_start_date), start_fy, sds_start_date)
+  ) %>%
```

Comment:
```suggestion
  mutate(sds_start_date = if_else(is.na(sds_start_date), as.Date(paste0(period, "-04-01")), sds_start_date)) %>%
```
Can just do this since we don't use `start_fy` anywhere else.